plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -95,6 +95,11 @@ type PartitionMetrics = {
     lastKnownCommit?: number
 }
 
+export interface TeamIDWithConfig {
```

Comment:
added with structure since if/when we start ingesting network performance events to ClickHouse we'll want the same control